test-utils/fcos.go

```diff
@@ -58,12 +62,13 @@ func (downloader *FcosDownload) DownloadImage() (string, error) {
 		}
 	}
 
+	uncompressedImage := ""
+	if uncompressedImage, err = Decompress(compressedImage); err != nil {
+		return "", err
+	}
 	if _, err := os.Stat(uncompressedImage); err == nil {
 		return uncompressedImage, nil
 	}
-	if err := Decompress(compressedImage, uncompressedImage); err != nil {
```

Comment:
Is it intentional that you are switching the `Stat` and the `Decompress` calls? If you don't want to skip decompression when the file already exists, I think the call to `Stat` can be removed?